### ISSUE TITLE
celocli governance:show --hotfix --whitelisters fails during key rotations

### ISSUE THREAD

**Issue by asaj** [celo-org/celo-monorepo#7631]
### Expected Behavior

We should use `signerToaccount` not `validatorSignerToAccount` in `currentValidatorAccountsSet`, otherwise, during a key rotation, the call will fail.

### Current Behavior

```
celocli governance:viewhotfix --hash 0x573b61ee8f642f6302cbc451270c5a6b986a3afd0a3dfca13588e4147ab2d37d --whitelisters
approved: false
executed: false
preparedEpoch: 0
passing: false
tally: 1
quorum: 67
    Error: Your request got reverted with the following reason string: not active authorized validator signer
```

**Comment by hqueue** [comment]
Also observed in `1.0.3` as below.

```
$ celocli governance:show --hotfix 0x573b61ee8f642f6302cbc451270c5a6b986a3afd0a3dfca13588e4147ab2d37d --whitelisters
 ›   Warning: @celo/celocli update available from 1.0.3 to 1.1.0.
approved: false
executed: false
preparedEpoch: 0
passing: false
tally: 6
quorum: 67
    Error: Your request got reverted with the following reason string: not active authorized validator signer
```

**Comment by aslawson** [comment]
@asaj this appears to happen only during a key rotation (I am able to run it now). 

just to understand its impact and importance here:

- how often do we feel there will be key rotations during a hotfix? 
- i'm a little less familiar with how this information may impact the hotfix process, but if it's important to be able to query for all the signers and this tool is the only simple way to do so, a malicious participant in theory could key rotate to obscure this information. wondering if this is a concern?

seems like a minor fix though and can group it with some other cli tooling updates 👍
